package.json

```diff
@@ -44,45 +44,45 @@
   },
   "homepage": "https://github.com/benmosher/eslint-plugin-import",
   "devDependencies": {
-    "babel-eslint": "8.0.x",
+    "babel-eslint": "^8.2.6",
     "babel-plugin-istanbul": "^4.1.6",
     "babel-preset-es2015-argon": "latest",
     "babel-register": "^6.26.0",
-    "babylon": "6.15.0",
+    "babylon": "^6.15.0",
     "chai": "^3.5.0",
-    "coveralls": "^3.0.0",
+    "coveralls": "^3.0.2",
     "cross-env": "^4.0.0",
     "eslint": "2.x - 5.x",
     "eslint-import-resolver-node": "file:./resolvers/node",
-    "eslint-import-resolver-typescript": "1.0.2",
+    "eslint-import-resolver-typescript": "^1.0.2",
     "eslint-import-resolver-webpack": "file:./resolvers/webpack",
     "eslint-module-utils": "file:./utils",
     "eslint-plugin-import": "2.x",
-    "gulp": "^3.9.0",
+    "gulp": "^3.9.1",
     "gulp-babel": "6.1.2",
-    "linklocal": "^2.6.0",
+    "linklocal": "^2.8.2",
     "mocha": "^3.5.3",
-    "nyc": "^11.7.1",
-    "redux": "^3.0.4",
-    "rimraf": "^2.6.2",
-    "sinon": "^2.3.2",
-    "typescript": "~2.8.1",
-    "typescript-eslint-parser": "^15.0.0"
+    "nyc": "^11.9.0",
+    "redux": "^3.7.2",
+    "rimraf": "^2.6.3",
+    "sinon": "^2.4.1",
+    "typescript": "^3.2.2",
+    "typescript-eslint-parser": "^21.0.2"
   },
   "peerDependencies": {
-    "eslint": "2.x - 5.x"
+    "eslint": "*"
```

Comment:
This change means that when eslint 6 comes out, we’ll claim compat with it, but won’t necessarily be. I think we should keep this line the same.

Reply:
That is fair, I forgot when I did this that peerDep mismatch is just a warning these days

Reply:
always has been, but `npm ls` will fail (which is good)